Source/EmbeddedBoundary/DistanceToEB.H

```diff
@@ -34,56 +34,95 @@ void normalize (amrex::RealVect& a) noexcept
                  a[2] *= inv_norm);
 }
 
+
+
+// This function calculates the normal vector using the nodes and cell-centered nodes
```

Comment:
```suggestion
// This function calculates the normal vector using the nodes and cell-centered data
```
Im guessing that should be nodal and cell-centered data (phi)?